index.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
+					<li><a href="" class="menu-link"><img src="./images/iconMenu.svg" alt="menu-icon"
+								class="menu-icon"></a></li>
+				</ul>
+				<div class="item-nav">
+					<ul>
+						<li class="nav-list"><a href="#" class="nav-link">Portfolio</a></li>
+						<li class="nav-list"><a href="#" class="nav-link">About </a></li>
+						<li class="nav-list"><a href="#" class="nav-link">contact</a></li>
+					</ul>
+				</div>
+			</nav>
+		</header>
+		<section class="presentation">
+			<div class="greeting">
+				<h1>I’m Mariane <br> Glad to see you!</h1>
+			</div>
+			<div class="self-description">
+				<p>
+					I’m a software developer! I can help you build a product , feature or website Look through some of
+					my work and
+					experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</div>
+			<a class="btn-mediaConnect" href="#">LET’S CONNECT</a>
+			<ul class="media-list">
+				<li>
+					<a href="" class="media-link"><img src="./images/twitterIcon.svg" alt="twitterIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/linkedIncon.svg" alt="linkdlnIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/mailIcon.svg" alt="mailIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon"
+							class="media-icon"></a>
+				</li>
+			</ul>
+		</section>
+
+		<!-- Project section-->
+
+		<section id="section-project">
+			<div class="details-card">
+				<div class="project-image1"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
```

Comment:
- [ ] Kindly apply the right `font-family` and `color` to this list of items. They are currently different from the Figma design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/220820504-f3787558-3035-48a5-8037-09e64db3a565.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/220820552-535ed86a-a7d9-4e4b-9d87-6e6484f5063d.png)